nipype/testing/data/fsl_motion_outliers_fd.txt

```diff
@@ -1,364 +1,364 @@
-0.130582 
-0.0712992 
-0.0355222 
-0.0534384 
-0.0404496 
-0.0728568 
-0.0347538 
-0.0365854 
-0.0236392 
-0.0343136 
-0.219582 
-0.133829 
-0.0631227 
-0.0636641 
-0.0814131 
-0.0479329 
-0.0353675 
-0.0384015 
-0.0310994 
-0.060522 
-0.043508 
-0.0305195 
-0.020338 
-0.0538009 
-0.0398593 
-0.044996 
-0.0778829 
-0.0675269 
-0.0303966 
-0.0684796 
-0.113632 
-0.173751 
-0.0953603 
-0.0384551 
-0.0348528 
-0.04526 
-0.0633001 
-0.356055 
-0.416511 
-0.253341 
-0.119676 
-0.139908 
-0.0971501 
-0.0554116 
-0.0258724 
-0.0459943 
-0.0833891 
-0.0624481 
-0.0657044 
-0.0389019 
-0.0205947 
-0.0409763 
-0.0498495 
-0.0416258 
-0.0166502 
-0.0353083 
-0.0997548 
-0.0965846 
-0.0235861 
-0.0633841 
-0.0154942 
-0.0357461 
-0.0800965 
-0.0650863 
-0.0211126 
-0.163237 
-0.125343 
-0.0540911 
-0.017874 
-0.0691969 
-0.11043 
-0.0835264 
-0.137328 
-0.118949 
-0.0753882 
-0.00275183 
-0.133472 
-0.244647 
-0.0847069 
-0.0577316 
-0.0751496 
-0.0750398 
-0.0972748 
-0.0748739 
-0.0458396 
-0.0310216 
-0.0366835 
-0.0356063 
-0.0553765 
-0.0639495 
-0.0650416 
-0.0505025 
-0.170091 
-0.13039 
-0.0472066 
-0.0572643 
-0.0815014 
-0.149822 
-0.227285 
-0.108031 
-0.0560778 
-0.0724768 
-0.0322847 
-0.11819 
-0.146627 
-0.111114 
-0.111445 
-0.020299 
-0.0410743 
-0.0268339 
-0.05399 
-0.0815168 
-0.0777002 
-0.0315828 
-0.165272 
-0.213802 
-0.0869493 
-0.036218 
-0.0332953 
-0.068719 
-0.0675272 
-0.0685225 
-0.0451177 
-0.119277 
-0.102194 
-0.0573171 
-0.0555505 
-0.115179 
-0.0471257 
-0.0575205 
-0.0487958 
-0.0594829 
-0.0896243 
-0.0537505 
-0.083843 
-0.0204785 
-0.0236064 
-0.075808 
-0.114648 
-0.0827696 
-0.0791115 
-0.102695 
-0.119031 
-0.0318184 
-0.0600644 
-0.0330951 
-0.107511 
-0.0710708 
-0.0392274 
-0.0742816 
-0.0839207 
-0.0467648 
-0.0345335 
-0.0146616 
-0.109282 
-0.187879 
-0.12107 
-0.0583698 
-0.0664069 
-0.0518948 
-0.0381672 
-0.0498546 
-0.0705965 
-0.0563436 
-0.0890585 
-0.106257 
-0.10538 
-0.140839 
-0.0393886 
-0.0322632 
-0.0641569 
-0.0545492 
-0.0362664 
-0.0250806 
-0.0621673 
-0.0157825 
-0.0509126 
-0.0748957 
-0.0965591 
-0.0749209 
-0.13329 
-0.132579 
-0.0985737 
-0.0772244 
-0.0360209 
-0.0445485 
-0.0692248 
-0.0694683 
-0.0347725 
-0.0302222 
-0.10371 
-0.158209 
-0.098906 
-0.0514085 
-0.0313609 
-0.0843055 
-0.0608005 
-0.063549 
-0.263942 
-0.17665 
-0.261261 
-0.0441281 
-0.0468925 
-0.0647964 
-0.044089 
-0.0629465 
-0.071587 
-0.029169 
-0.0401875 
-0.026996 
-0.0610415 
-0.0458035 
-0.035757 
-0.0468055 
-0.0355955 
-0.0530505 
-0.204683 
-0.116196 
-0.0907005 
-0.066667 
-0.124328 
-0.067071 
-0.09391 
-0.0647426 
-0.095498 
-0.140501 
-0.105467 
-0.0573551 
-0.0305451 
-0.026572 
-0.037701 
-0.0364285 
-0.0338849 
-0.0278505 
-0.0145665 
-0.109527 
-0.133339 
-0.0582865 
-0.0516891 
-0.0618162 
-0.103562 
-0.0196925 
-0.0693495 
-0.137835 
-0.0963195 
-0.0614775 
-0.0551995 
-0.0580145 
-0.034866 
-0.0435379 
-0.0189935 
-0.0270825 
-0.0893485 
-0.0466895 
-0.048314 
-0.0224355 
-0.10331 
+0.0922165
```

Comment:
These files were modified when I ran make check-before-commit. Since they were automatically generated by the tool, I assumed it served a purpose, but I can't be sure.